screenshot-app/pages/index.js

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+import { useState } from "react";
+
+export default function Home() {
+  const [url, setUrl] = useState(null);
+  const [response, setResponse] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const getScreenshot = async () => {
+    try {
+      setLoading(true);
+      const res = await axios.get("api/screenshot/", {
+        params: { url },
+      });
+      const { data } = res;
+      setLoading(false);
+      setResponse(data.screenshotUrl);
+    } catch (error) {
+      setLoading(false);
```

Comment:
It would be good to show an error message when the API fails to load.

Reply:
Sure.

Reply:
Let me know once this is done and I will merge the PR.

Reply:
I have made the changes, added error handling and tested it.